enterprise/frontend/src/embedding-sdk/components/private/SdkGlobalStylesWrapper.tsx

```diff
@@ -1,54 +0,0 @@
-import { css } from "@emotion/react";
```

Comment:
Global styles were the reason of the issues so I removed this component and only added the few bits we needed (for fonts) to the provider